.github/workflows/lint.yaml

```diff
@@ -1,7 +1,7 @@
 name: Lint
 
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
why do we need this to be pull_request_target?

Reply:
So that they don't require approval for seeing linting  issue

Reply:
For forked repo PRs, workflows with `pull_request` get repo `read` permission while workflows triggered with `pull_request_target` get repo `read/write`  permission. The `pull_request_target` should not be used  for workflows that can commit the code repo.  Is this workflow only points out linting issues or it fixes them as well via code commit? If it is pushing code then we should not use `pull_request_target`.

Reply:
It fixes them as well but doesn't commit or push the changes. Changes are made to local runner git repo

Reply:
Have made it back to `pull_request`

Reply:
back to `pull_request_target`